src/api/api.ts

```diff
@@ -113,20 +113,15 @@ class API {
     if (filters.voivodeship) {
       queryParams.append('voivodeship', filters.voivodeship);
     }
-
-    // if (filters.amenities && filters.amenities.length > 0) {
-    //   filters.amenities.forEach((amenity) => {
-    //     queryParams.append('offerSearchCriteria.amenity', amenity);
-    //   });
-    // }
   
     const queryString = queryParams.toString();
+    
     const requestBody = filters.animals?.map((animal) => ({
       animalType: animal.animalType,
       offerConfigurations: animal.offerConfigurations.map((offer) => ({
         ...offer,
         minPrice: offer.minPrice ? offer.minPrice : 0.01,
-        maxPrice: offer.maxPrice ? offer.maxPrice : 99999
+        maxPrice: offer.maxPrice ? offer.maxPrice : 99999.99
```

Comment:
Skoro nie dajemy użytkownikowi możliwości filtrowania ceny z kropką/przecinkiem, to może ograniczyć ceny od 0 do 99999?

Reply:
Też wsm maksymalna cena ma 5 cyfr. A tak trochę jeszcze z innej strony, jeżeli opiekun może wystawić ofertę z ceną po przecinku, np. 99999.99, a klient może filtrować bez przecinka i do 5 cyfr, to taka (w prawdzie mało prawdopodobna) cena oferty nie wyświetliłaby się klientowi przy filtrze. Chociaż to taki szczegół

Reply:
No racja, chyba najlepiej byłoby dać możliwość filtrowania z dwoma miejscami po przecinku, chociaż z drugiej strony pewnie mało który użytkownik bylby tak dokładny.